samples/kubernetes/end2end/docs/existing-domain.md

```diff
@@ -0,0 +1,36 @@
+## Monitoring an Existing WebLogic Domain
+In this step, we show you how to integrate Prometheus and Grafana with an existing WebLogic domain.
+
+Assume you have already followed [Manage Domains](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/) guide in the WebLogic Kubernetes Operator to run a WebLogic domain and have followed previous steps in this sample to setup Prometheus and Grafana.
+
+### Installing the wls-exporter to the WebLogic domain
+You need to install the wls-exporter web application to the WebLogic domain.
+
+First, generate the wls-exporter war file following the [building from source](https://github.com/oracle/weblogic-monitoring-exporter#building-from-source) guide using the configuration file in this sample [exporter-config.yaml](../dashboard/exporter-config.yaml). Then deploy the war to the WebLogic servers and clusters using WLST or WDT.  
```

Comment:
war file -> WAR file 
Then deploy the war to the WebLogic servers -> Then deploy the WAR file to the WebLogic Servers